ci_requirements.txt

```diff
@@ -1 +1 @@
-pulp_file
+pulp_file<1.15
```

Comment:
We're choosing <1.15 because we know that file/1.15+ has "problems" with some of the test-harness-changes in core>3.40, yeah?

Reply:
Yes, experiment showed that requesting `pulp_file<1.16` did not solve the issue of missing `pulp_file` test utility functions.
I needed we can try to find the "last 1.15.z" still supporting the tests here, but I believe that puts us in a corner too.